cogs/tags.py

```diff
@@ -1,54 +1,79 @@
-from discord.ext import commands
-import discord
-
 import asyncio
+from datetime import datetime
+
 import asyncpg
+import discord
+from config import TAGS_LOG_CHANNEL_ID
+from discord.ext import commands
 
+from .utils.checks import is_admin, is_engineer_check, is_staff
 from .utils.DataBase.tag import Tag
-from .utils.checks import is_engineer_check, is_admin, is_staff
+
+EMOJIS = [
+    "\N{WHITE HEAVY CHECK MARK}",
+    "\N{CROSS MARK}",
+]
 
 
 def setup(bot):
     bot.add_cog(TagCommands(bot=bot))
 
 
 class TagCommands(commands.Cog, name="Tags"):
-    def __init__(self, bot):
+    def __init__(self, bot: commands.Bot):
         self.bot = bot
+        self.log_channel = self.bot.get_channel(TAGS_LOG_CHANNEL_ID)
 
     def cog_check(self, ctx):
         if ctx.guild is None:
             return False
+
         return True
 
+    ####################################################################################################################
+    # Commands
+    ####################################################################################################################
+
     @commands.group(invoke_without_command=True)
     async def tag(self, ctx, *, name: lambda inp: inp.lower()):
         """Main tag group."""
+        name = await commands.clean_content().convert(ctx=ctx, argument=name)
```

Comment:
I think `name: commands.clean_content` then `name = name.lower()` would be clearer than invoking the converter for clean content.